src/containers/Routes.jsx

```diff
@@ -243,55 +193,15 @@ const Routes = () => {
         path="/campaigns/:id/traces"
         render={({ match }) => <Redirect to={`/campaigns/${match.params.id}`} />}
       />
-      <Route
-        exact
-        path="/donations"
-        render={props => (
-          <Donations
-            key={currentUser ? currentUser.id : 0}
-            currentUser={currentUser}
-            balance={balance}
-            {...props}
-          />
-        )}
-      />
-      <Route
-        exact
-        path="/delegations"
-        render={props => (
-          <Delegations
-            key={currentUser ? currentUser.id : 0}
-            currentUser={currentUser}
-            balance={balance}
-            {...props}
-          />
-        )}
-      />
+      <Route exact path="/donations" render={() => <Donations />} />
+      <Route exact path="/delegations" render={() => <Delegations />} />
       <Route
         exact
         path="/my-communities"
-        render={props => (
-          <MyCommunities
-            key={currentUser ? currentUser.id : 0}
-            currentUser={currentUser}
-            balance={balance}
-            {...props}
-          />
-        )}
+        render={() => <MyCommunities currentUser={currentUser} balance={balance} />}
```

Comment:
Let's remove currentUser and balance from these components too, it means making them Hook Base components